api/application_msgraph.go

```diff
@@ -108,23 +133,19 @@ func (c *AppClient) DeleteApplication(ctx context.Context, applicationObjectID s
 		return autorest.NewErrorWithError(err, "provider", "DeleteApplication", resp, "Failure sending request")
 	}
 
-	err = autorest.Respond(
+	return autorest.Respond(
 		resp,
 		c.client.ByInspecting(),
 		azure.WithErrorUnlessStatusCode(http.StatusNoContent, http.StatusNotFound),
 		autorest.ByClosing())
-
 	if err != nil {
```

Comment:
This code is unreachable with the return on L136.

Reply:
This was due to a bad rebase. Good catch!